docs/source/tutorials_source/package/Base-RCNN-FPN.yaml

```diff
@@ -0,0 +1,42 @@
+MODEL:
```

Comment:
This config here does not differ from the official one from detectron2 (see https://github.com/facebookresearch/detectron2/blob/main/configs/Base-RCNN-FPN.yaml). Can we drop ours? and add somewhere the link to the exact config we use?

Reply:
It says in the text that you need to download the config file. I just downloaded it permanently so the tutorial runs through.